echopype/tests/commongrid/test_api.py

```diff
@@ -0,0 +1,164 @@
+def test_compute_MVBS_w_latlon(ds_Sv_er_regular_w_latlon, lat_attrs, lon_attrs):
+    """Testing for compute_MVBS with latitude and longitude"""
+    from echopype.consolidate.api import POSITION_VARIABLES
+    ds_MVBS = ep.commongrid.compute_MVBS(
+        ds_Sv_er_regular_w_latlon,
+        range_meter_bin=5,
+        ping_time_bin="10S"
+    )
+    for var in POSITION_VARIABLES:
+        # Check to ensure variable is in dataset
+        assert var in ds_MVBS.data_vars
+        # Check for correct shape, which is just ping time
+        assert ds_MVBS[var].shape == ds_MVBS.ping_time.shape
+        
+        # Check if attributes match
+        if var == "latitude":
+            assert ds_MVBS[var].attrs == lat_attrs
+        elif var == "longitude":
+            assert ds_MVBS[var].attrs == lon_attrs
+        
+    
+@pytest.mark.unit
+@pytest.mark.parametrize("range_var", ["my_range", "echo_range", "depth"])
+def test_compute_MVBS_invalid_range_var(ds_Sv_er_regular, range_var):
+    """Test compute MVBS range_var on mock data"""
+
+    if range_var == "my_range":
+        with pytest.raises(
+            ValueError,
+            match="range_var must be one of 'echo_range' or 'depth'."
+        ):
+            ep.commongrid.compute_MVBS(ds_Sv_er_regular, range_var=range_var)
+    elif range_var == "depth":
+        with pytest.raises(
+            ValueError,
+            match=f"range_var '{range_var}' does not exist in the input dataset."
+        ):
+            ep.commongrid.compute_MVBS(ds_Sv_er_regular, range_var=range_var)
+    else:
+        pass
+
+@pytest.mark.integration
+def test_compute_MVBS(test_data_samples):
+    """
+    Test running through from open_raw to compute_MVBS.
+    """
+    (
+        filepath,
+        sonar_model,
+        azfp_xml_path,
+        range_kwargs,
+    ) = test_data_samples
+    ed = ep.open_raw(filepath, sonar_model, azfp_xml_path)
+    if ed.sonar_model.lower() == 'azfp':
+        avg_temperature = ed["Environment"]['temperature'].values.mean()
+        env_params = {
+            'temperature': avg_temperature,
+            'salinity': 27.9,
+            'pressure': 59,
+        }
+        range_kwargs['env_params'] = env_params
+        if 'azfp_cal_type' in range_kwargs:
+            range_kwargs.pop('azfp_cal_type')
+    Sv = ep.calibrate.compute_Sv(ed, **range_kwargs)
+    ping_time_bin = "20S"
+    ds_MVBS = ep.commongrid.compute_MVBS(Sv, ping_time_bin=ping_time_bin)
+    assert ds_MVBS is not None
+    
+    # Test to see if ping_time was resampled correctly
+    expected_ping_time = (
+        Sv["ping_time"]
+        .resample(ping_time=ping_time_bin, skipna=True)
+        .asfreq()
+        .indexes["ping_time"]
+    )
+    assert np.array_equal(ds_MVBS.ping_time.data, expected_ping_time.values)
+
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    ("er_type"),
+    [
+        ("regular"),
+        ("irregular"),
+    ],
+)
+def test_compute_MVBS_er_output(request, er_type):
```

Comment:
This should probably be changed to make it less cryptic.
```suggestion
def test_compute_MVBS_range_output(request, er_type):
```